easybuild/easyconfigs/p/Python/Python-3.5.1-intel-2016a.eb

```diff
@@ -0,0 +1,112 @@
+name = 'Python'
+version = '3.5.1'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.10.4'
+scipyversion = '0.17.0'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+#   ('OpenSSL', '1.0.1q'),  # OS dependency should be preferred if the os version is more recent then this version, it's
+#   nice to have an up to date openssl for security reasons
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev')]
+
+# order is important!
+# package versions updated Feb 25th 2016
+exts_list = [
+    ('setuptools', '20.1.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    ('pip', '8.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pip/'],
+    }),
+    ('nose', '1.3.7', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/nose/'],
+    }),
+    ('numpy', numpyversion, {
+        'source_urls': ['https://pypi.python.org/packages/source/n/numpy/'],
+        'patches': [
+            'numpy-1.8.0-mkl.patch', # % numpyversion,
+        ],
+    }),
+    ('scipy', scipyversion, {
+        'source_urls': ['https://pypi.python.org/packages/source/s/scipy/'],
+    }),
+    ('blist', '1.3.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/blist/'],
+    }),
+    ('mpi4py', '2.0.0', {
+        'source_urls': ['http://bitbucket.org/mpi4py/mpi4py/downloads/'],
+    }),
+    ('paycheck', '1.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/paycheck/'],
+        'patches': [
+            'paycheck-1.0.2_setup-open-README-utf8.patch',
+        ],
+    }),
+    ('pbr', '0.11.1', {
```

Comment:
apparently the latest version here is 1.8.1 (and the lockfile below needs it)

The default version you see on PyPI is wrong, cfr. https://pypi.python.org/pypi/pbr/1.8.1 and https://launchpad.net/pbr/+series